cmd/validate/image.go

```diff
@@ -127,6 +132,14 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 
 			Write output in YAML format to stdout and in HACBS format to a file
 			  ec validate image --image registry/name:tag --output yaml --output hacbs=<path>
+
+			Validate a single image with keyless workflow. This is an experimental feature
+			that requires setting the EC_EXPERIMENTAL environment variable to "1".
+
+			  EC_EXPERIMENTAL="1" ec validate image --image registry/name:tag --policy my-policy \
+			  --certificate-identity-regexp '.*' \
+			  --certificate-oidc-issuer-regexp '.*' \
```

Comment:
Suggestion: One more example that doesn't use `'.*'`, just so reader can see how a tighter regex might look. E.g. should there be a '^' at the start or is it implicit?